docs/writing-docs/introduction.md

```diff
@@ -4,21 +4,12 @@ title: 'How to document components'
 
 When you write component stories during development, you also create basic documentation to revisit later.
 
-Storybook gives you tools to expand this basic documentation with prose and layout that feature your components and stories prominently. That allows you to create UI library usage guidelines, design system sites, and more.
+Storybook gives you tools to expand this essential documentation with prose and layout that feature your components and stories prominently. That allows you to create UI library usage guidelines, design system sites, and more.
 
-<video autoPlay muted playsInline loop>
-  <source
-    src="addon-docs-optimized.mp4"
-    type="video/mp4"
-  />
-</video>
+![Storybook Docs UI](./storybook-docs-ui-optimized.png)
```

Comment:
This image doesn't actually show Docs (aside from the orange icons in the sidebar).

Reply:
I know, and I went with this one as a placeholder for two reasons:
1- Show off the unified UI
2- We actually don't have that many examples showcasing the feature yet. Once we do this one will be updated.